README.md

```diff
@@ -40,3 +40,22 @@ Here is an example program that uses this module to add more context to errors:
         console.log(newErr.stack);
     });
 ```
+
+How to inherit
+--------------
+
+It is recomennded to use explicit names for Error classes. You can do it like this:
+
+```js
+var util = require('util');
+var objectAssign = require('object-assign');
+var NestedError = require('nested-error-stacks');
+
+function MyError(message, nested) {
+    NestedError.call(this, message, nested);
+    objectAssign(this, nested); // copy error.code and other fields
```

Comment:
This particular line seems very subjective.  In most of my uses, I would not want to do this.  Would you mind if I took the example without the use of object-assign?